pkg/filesystem/workdir.go

```diff
@@ -1,34 +0,0 @@
-// Copyright 2023 The Okteto Authors
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package filesystem
-
-import "path/filepath"
-
-// GetWorkdirFromManifestPath sets the path
-func GetWorkdirFromManifestPath(manifestPath string) string {
```

Comment:
I'm wondering if we should be using this one instead of the one within `model` package. It seems the implementation is the same and it would help us to reduce the footprint of model package along the codebase

Reply:
good idea, updated ✅